optuna/exceptions.py

```diff
@@ -13,22 +13,19 @@ class TrialPruned(OptunaError):
 
     Example:
 
-        .. testsetup::
+        .. testcode::
 
+            import optuna
             import numpy as np
+            from sklearn.linear_model import SGDClassifier
+
+            from sklearn.datasets import load_iris
             from sklearn.model_selection import train_test_split
```

Comment:
Please remove the unnecessary blank line and sort lines in alphabetical order.
```suggestion
            from sklearn.datasets import load_iris
            from sklearn.linear_model import SGDClassifier
            from sklearn.model_selection import train_test_split
```